core/api/node_api.hpp

```diff
@@ -140,7 +143,8 @@ namespace fc::api {
   };
 
   struct MinerPower {
-    Claim miner, total;
+    Claim miner;
+    Claim total;
```

Comment:
```suggestion
    Claim total;
    bool has_min_power;
```
json name "HasMinPower"

Reply:
It turns out that this is not necessary